### DISCUSSION TITLE
Issue assigned by someone else shows as self-assigned on IOS

### DISCUSSION THREAD

**Discussion by beg1c** [community/community#110531] [Mobile] [answered]
### Select Topic Area

Bug

### Body

On IOS, issue assignment shows as self-assigned even if it is assigned by someone else. Images below are for same issue assignment, shown on desktop and on mobile.

In browser:
![Snimka zaslona 2024-02-29 170128](https://github.com/community/community/assets/72312119/e76c4035-9c5a-4db4-9038-62a691b9e6c2)

Mobile IOS: 
![IMG_9533](https://github.com/community/community/assets/72312119/31c9a24b-25dc-4cac-a9e3-fd79eebfeb62)

**Comment by stevepopovich** [comment]
Hi @beg1c is this consistent for you? Do you have an issue where you can easily reproduce this? I am having trouble reproing

**Comment by beg1c** [comment]
Hi @stevepopovich, weirdly it works normal now. Below are screenshots from few hours ago and now. For other assignees it also appeared as self-assignment. For example to me it would appear like "stevepopovich self-assigned this" instead of "bob assigned stevepopovich". I will give updates if I manage to reproduce it. 

<img src="https://github.com/community/community/assets/72312119/8c20c85e-f158-4c8a-bfb7-15c59f668c2d" width="300"/>

<img src="https://github.com/community/community/assets/72312119/eaba4193-03bc-48f9-807f-3206641941f6" width="300"/>

**Reply by stevepopovich** [reply, marked as answer]
That's...super weird!! There is nothing are on end (like a recent outage or incident) that explains this quickly. So I will file this with the team and we will keep an eye on this. If you come up with consistent repro, **please** write in again. I will give this to the iOS pros who will try as well.

Thanks for writing in!
